go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	k8s.io/apimachinery v0.18.3
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6
+	k8s.io/utils v0.0.0-20200414100711-2df71ebbae66
```

Comment:
This change is not related to this Pull Request. Please remove it.

Reply:
Agreed. Removing the same.